src/main/java/Display.java

```diff
@@ -0,0 +1,5 @@
+public interface Display {
+
+    public void sendMessageToDisplay(String message);
```

Comment:
You have two verbs in this method name which make me wonder if it couldn't be improve. Like for example: displayMessage, but the interface name is already *Display* so may be something like: printMessage.

As I'm still learning about naming stuff, I'm curious about your opinion about it! :)